dev_envs/mac-env-setup.md

```diff
@@ -43,12 +43,6 @@ sudo xcodebuild -license
 xcode-select --install
 ```
 
-### Docker ###
-
-Download the latest stable [Docker for Mac](https://www.docker.com/docker-mac).
-Install it by double-clicking the downloaded `dmg` file and dragging the
-Docker application file to the `Applications` folder.
-
```

Comment:
Why not leave this section?  Some folks will be OK with installing Docker for Mac.

Reply:
Is our team still the target audience for this document?  If so, I don't think we need the Docker for Mac section anymore.  If we ever purchase Docker, we can always add it back.

If the target audience is wider than our team, then I agree that we can leave it, maybe adding a sentence about the licensing.

Reply:
I was structuring it like @dav3r was suggesting. I can also see @jsf9k's point since the project is OSS and in an ideal situation outside contributions is encouraged. The question I have is are the components that they would use Docker for exclusive to CISA VPN's, AWS accounts, etc.? If that is the case then I suppose it wouldn't really matter if they had Docker or Colima if they can't really make any of the components work with it unless they just want to use it for their own sandbox environment type stuff.

Reply:
@dav3r and @jsf9k any consensus? I am leaning towards leaving it in but just wanted to check with you all for final decision.

Reply:
I vote for leaving it in, but adding an italicized paragraph at the very beginning of the section that states _Note that [Colima](link_to_Colima_section) is our preferred solution on OSX._

Reply:
Also note that the download is now called "Docker Desktop on Mac".  Note the addition of "Desktop" _and_ the modified preposition.

Reply:
> I vote for leaving it in, but adding an italicized paragraph at the very beginning of the section that states _Note that [Colima](link_to_Colima_section) is our preferred solution on OSX._

I'm fine with that approach, but will note that "macOS" is the current modern nomenclature vs. "OSX".

Reply:
@jsf9k and @dav3r I made the fix here: https://github.com/cisagov/development-guide/pull/96/commits/a7d7d8e99dc3f1bab1c450aee87e9d8bfb4c39cc

Reply:
See [this tweak suggestion](https://github.com/cisagov/development-guide/pull/96#discussion_r1497979493).

Reply:
@jsf9k consider it done

Reply:
@jsf9k - Can this conversation be resolved?